tests/index.test.js

```diff
@@ -0,0 +1,20 @@
+import chai from 'chai';
+import chaitHttp from 'chai-http';
+import app from '../index';
+
+const should = chai.should();
+
+chai.use(chaitHttp);
+
+// testing the root rout
+describe('Root test', () => {
+    it('should pass as the response is json object and with message', (done) => {
```

Comment:
I would be better if you express the `it` block in terms of expected return as shown below:
```javascript
     it('returns a response with status code 200 and JSON object', (done) => {...});
```